lib/LocalDevManager.ts

```diff
@@ -205,11 +249,11 @@ class LocalDevManager {
         getProjectDetailUrl(
```

Comment:
I wonder why `getProjectDetailUrl` returns if the projectName doesn't exist. The projectName field is required

Reply:
I think that was left over from before it was typed. Once we're sure everywhere in the CLI is passing in a string, we can probably remove that check